README.md

```diff
@@ -52,35 +44,31 @@ It is distributed as a separate npm package: [https://www.npmjs.com/package/node
 
 ## Testing
 
-In addition to running the tests in the converted modules we also have
-
-* a converted version of the NAN examples
+In addition to running the tests in the converted modules we also have a converted version of the NAN examples
   [node-addon-examples](https://github.com/nodejs/node-addon-examples)
 
-* a converted version of the [core addons tests](https://github.com/nodejs/node/tree/master/test/addons-napi) which can be run with `make test addons-napi`
-
 ## How to get involved
-* Convert a native module to use [N-API](https://github.com/nodejs/abi-stable-node/blob/api-prototype-8.x/src/node_api.h) and report issues on conversion and performance;
+* Convert a native module to use [Node-API](https://github.com/nodejs/abi-stable-node/blob/api-prototype-8.x/src/node_api.h) and report issues on conversion and performance;
 * Port ABI stable APIs to your fork of Node and let us know if there are gaps;
 * Review the [roadmap](https://github.com/nodejs/abi-stable-node/milestones) and see how you
 can help accelerate this project.
 
 ## Badges
 
-The use of badges is recommended to indicate the minimum version of N-API
+The use of badges is recommended to indicate the minimum version of Node-API
 required for the module. This helps to determine which Node.js major versions
-are supported. Addon maintainers can consult the [N-API support matrix][] to
-determine which Node.js versions provide a given N-API version. The following
+are supported. Addon maintainers can consult the [Node-API support matrix][] to
+determine which Node.js versions provide a given Node-API version. The following
 badges are available:
 
-![N-API v1 Badge](assets/N-API%20v1%20Badge.svg)
-![N-API v2 Badge](assets/N-API%20v2%20Badge.svg)
-![N-API v3 Badge](assets/N-API%20v3%20Badge.svg)
-![N-API v4 Badge](assets/N-API%20v4%20Badge.svg)
-![N-API v5 Badge](assets/N-API%20v5%20Badge.svg)
-![N-API v6 Badge](assets/N-API%20v6%20Badge.svg)
-![N-API v7 Badge](assets/N-API%20v7%20Badge.svg)
-![N-API Experimental Version Badge](assets/N-API%20Experimental%20Version%20Badge.svg)
+![Node-API v1 Badge](assets/N-API%20v1%20Badge.svg)
```

Comment:
I was assuming we can put `assets/Node-API%20v1%20Badge.svg` along side with existing `assets/N-API%20v1%20Badge.svg` (those are changed with "Node-API" label too)?